app/assets/stylesheets/content/_timelines.scss

```diff
@@ -125,7 +125,7 @@ table.timelines-dates {
     }
   }
   &.timelines-pt td {
-    &.timelines-pt-reorder, &.timelines-pt-allows_association, &.timelines-pt-actions {
+    &.timelines-pt-reorder, &.timelines-pt-allows_association {
```

Comment:
At least the part regarding "each on its own line" sounds legit to me.

@myabc What do you say?